sources/packages/web/src/components/generic/SnackBar.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <v-snackbar
+    v-model="showSnackBar"
+    variant="text"
+    :color="snackBarType"
+    location="top right"
+    :timeout="snackBarTimeOut"
+    :class="snackBarClass"
+    class="mt-16 mr-8"
+  >
+    <div class="mt-2">
+      <v-row>
+        <v-col cols="1">
+          <v-icon
+            :icon="snackBarIcon"
+            class="icon-color mt-0"
```

Comment:
The icon seems a bit too down, can we just try to set a negative margin like `mt-n1`?

Reply:
as discussed, the icons are aligned correctly in my local, 
![image](https://user-images.githubusercontent.com/77353155/180507658-c0e61cde-75c7-45a6-80eb-0d06cba565d8.png)
I am resolving the comment